src/Sulu/Bundle/HttpCacheBundle/Cache/SuluHttpCache.php

```diff
@@ -34,16 +39,21 @@ class SuluHttpCache extends HttpCache implements CacheInvalidation
      * @param HttpKernelInterface $kernel
      * @param string $cacheDir
      */
-    public function __construct(HttpKernelInterface $kernel, $cacheDir = null)
+    public function __construct(SuluKernel $kernel, $cacheDir = null)
```

Comment:
would keep here `HttpKernelInterface` and inject `%sulu.common_cache_dir%` so its possible to change the directory for this service only else you are not able to cahnge it only for this service.